README.md

```diff
@@ -57,7 +57,7 @@ To connect to the RisingWave server, you will need to [install PostgreSQL shell]
 
 ```shell
 # Use psql to connect RisingWave cluster
-psql -h localhost -p 4566 -d dev
+psql -h localhost -p 4566 -d dev -U risingwave
```

Comment:
Why not "root"? It's more intuitive to me, and shorter ☺️

Reply:
Whatever, and materialize uses materialize (

Reply:
Both are ok. Well, shorter will be more convenient. I will use root instead. 😊